### PR TITLE
Vylepšení znalostní báze pro DPNK 2020

### PR DESCRIPTION
**Pull request by PetrDlouhy**

### CODE REVIEW

File: helpdesk/views/kb.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def category(request, slug, iframe=False):
     category = get_object_or_404(KBCategory, slug__iexact=slug)
     if not user.huser_from_request(request).can_access_kbcategory(category):
         raise Http404
-    items = category.kbitem_set.all()
+    items = category.kbitem_set.filter(enabled=True)
```
Comment (timthelion) [inline helpdesk/views/kb.py:33]: https://github.com/auto-mat/django-helpdesk/blob/teams/helpdesk/forms.py#L187

this line needs to be updated to filter out disabled items too.

Comment (PetrDlouhy) [reply helpdesk/views/kb.py:33]: Do you mean `KBCategory` with `public=False`?

Comment (timthelion) [reply helpdesk/views/kb.py:33]: No, I mean that in the Ticket form, you can choose the kbitem that the ticket belongs to. That choice field (which I linked to) should not include kbitems which are not enabled.

Comment (PetrDlouhy) [reply helpdesk/views/kb.py:33]: @timthelion Fixed.

**Pull request merged**